src/SwedbankPay.Sdk.Tests/PaymentTests/SwishPaymentTests.cs

```diff
@@ -22,8 +22,9 @@ public async Task AbortPayment()
             var payment = await this.Sut.Payment.CreateSwishPayment(paymentRequest);
 
             Assert.NotNull(payment);
+            Assert.NotNull(payment.Operations.Abort);
 
-            var paymentResponseContainer = await payment.Operations.Abort.Invoke();
+            var paymentResponseContainer = await payment.Operations.Abort();
```

Comment:
Abort could be null. Nullcheck should still exist

Reply:
If it is null then the `Assert.NotNull(payment.Operations.Abort);` should fail.

Is the usage of the operations not to be like this:
```
var paymentOrder = await _swedbankPayClient.PaymentOrder.Get(paymentOrderUri));
if (paymentOrder.Operations.Abort != null)
{
    await paymentOrder.Operations.Abort();
}
```

Reply:
Sorry, i didn't notice you added `Assert.NotNull(payment.Operations.Abort);` 
In this case Abort could not be null then.

You could do it like above, or just use the null-conditional operator and check if response is null.